health-sdk/sdk/src/main/java/net/gini/android/health/sdk/paymentcomponent/PaymentComponentView.kt

```diff
@@ -42,7 +42,9 @@ class PaymentComponentView(context: Context, attrs: AttributeSet?) : ConstraintL
             }
         }
 
-    private val binding = GhsViewPaymentComponentBinding.inflate(getLayoutInflaterWithGiniHealthTheme(), this)
+    private val binding = GhsViewPaymentComponentBinding.inflate(getLayoutInflaterWithGiniHealthTheme(), this).also {
+        it.ghsSelectBankPicker.root.background = context.getDrawable(R.drawable.ghs_bordered_background)
```

Comment:
[Request] AndroidStudio recommended not to use `context.getDrawable()` directly and we should make sure we apply our theme to the context:
```suggestion
        it.ghsSelectBankPicker.root.background = ContextCompat.getDrawable(context.wrappedWithGiniHealthTheme(), R.drawable.ghs_bordered_background)
```

Reply:
Thank you for pointing this out - I switched to using wrapped context.